examples/yaml/control_saveandread.yaml

```diff
@@ -0,0 +1,117 @@
+# General Description:  
+# Any key that is specific for a plot type will begin with ts for timeseries, ty for taylor
+# Opt: Specifying the variable or variable group is optional
+# For now all plots except time series average over the analysis window. 
+# Seting axis values - If set_axis = True in data_proc section of each plot_grp the yaxis for the plot will be set based on the values specified in the obs section for each variable. If set_axis is set to False, then defaults will be used. 'vmin_plot' and 'vmax_plot' are needed for 'timeseries', 'spatial_overlay', and 'boxplot'. 'vdiff_plot' is needed for 'spatial_bias' plots and'ty_scale' is needed for 'taylor' plots. 'nlevels' or the number of levels used in the contour plot can also optionally be provided for spatial_overlay plot. If set_axis = True and the proper limits are not provided in the obs section, a warning will print, and the plot will be created using the default limits.
+analysis:
+  start_time: '2019-09-05-00:00:00' #UTC
+  end_time: '2019-09-06-00:00:00' #UTC
+  output_dir: /wrk/charkins/melodies_monet/aircraft/analysis_saveout/example #Opt if not specified plots stored in code directory.
+  debug: True
+  save:
+      paired:
+          method: 'netcdf' # 'netcdf' or 'pkl'
+          prefix: '0905' # use only with method=netcdf
+          # output_name: '0905.pkl' # use only with method=pkl
+          data: 'all'   # 'all' to save out all pairs or ['pair1','pair2',...] to save out specific pairs. With method='pkl' this is ignored and always saves all.
+      # models:
+      # obs:
+  read:
+      paired:
+          method: 'netcdf' # 'netcdf' or 'pkl'
+          filenames: {'airnow_wrfchem_v4.2':['0905_airnow_wrfchem_v4.2.nc4']} # example for netcdf method. Uses dict of form {group1: str or iterable of filenames, group2:...}
```

Comment:
I guess my comment above is really does it default to using the output_dir to read in the output and if so, why does the ReadTheDocs example look different and include the entire directory name?

Reply:
So currently it will always save to output_dir but reading I left such that it can be read from anywhere. In this example it is reading from the code directory and in the prior comment it is reading from the output_dir that was set in the yaml file, but with a relative address.

I like your solution of having optional 'output_dir_read' and 'output_dir_write' options and then defaulting to output_dir if they are not specified.

My only concern with defaulting to output_dir if output_dir_read is not specified that it limits the user to reading files from a single directory. I could add the ability to do something like `output_dir_read: None`, which would ignore the default directory when reading and instead read directly from the path specified in filenames: .

Reply:
I like this. Let's add this additional capability that if you can set output_dir_read = None, then you can just add the file paths separately for each filename. Let's just explain this in the yaml description in the docs since it's a bit complicated.

Reply:
As it turns out, you need to set it as `output_dir_read: null` in the yaml file so that python reads it as `output_dir_read = None` in the `control_dict`. I'll explain in the docs.